.github/workflows/build-terraform-oss.yml

```diff
@@ -0,0 +1,101 @@
+---
+name: build_terraform
+
+# This workflow is intended to be called by the build workflow for each Consul
+# binary that needs to be built and packaged. The crt make targets that are
+# utilized automatically determine build metadata and handle building and
+# packing consul.
+
+on:
+  workflow_call:
+    inputs:
+      cgo-enabled:
+        type: string
+        default: 0
```

Comment:
I lean towards making this required as well, so we're forced to be explicit about including it in the matrix definition for any new targets - what do you think?

Reply:
Yeah, that seems like a sensible move 😄